source/shared/net/guides/multisig-transfer.rst

```diff
@@ -1,9 +1,9 @@
 
 .. _create-multisig-scheduled:
 
-=====================================================================
-Create a scheduled multi-signature GTU transfer in the Desktop Wallet
-=====================================================================
+=================================================================
+Create a scheduled multi-signature transfer in the Desktop Wallet
```

Comment:
I'm not sure why the focus on multi-signature is necessary.

The fact that a transfer is with a schedule, and the fact that it is a multi-signature transfer are orthogonal concepts.